### ISSUE TITLE
Need doc for disabled field

### ISSUE THREAD

**Issue by JustinKuli** [stolostron/policy-generator-plugin#47]
In the reference YAML, under `policies`, the disabled field comment refers to a non-existent section in the `policyDefaults`: https://github.com/stolostron/policy-generator-plugin/blob/main/docs/policygenerator-reference.yaml#L142

I'm not sure if that field should also be exposed in the defaults? But either way, it should be explained somewhere.

**Comment by mprahl** [comment]
@JustinKuli you're right. `disabled` isn't available in the policy defaults in the code. It probably should be to be consistent with the rest of the fields.

What do you think @dhaiducek?

**Comment by dhaiducek** [comment]
Oh, wow--nice catch, @JustinKuli! Should be a straightforward change--I agree it should be consistent.

**Comment by dhaiducek** [comment]
Could we please migrate this issue to `backlog` for sprint tracking, though?
